drivers/bus/isapnp/CMakeLists.txt

```diff
@@ -1,13 +1,22 @@
 
-list(APPEND SOURCE
-    isapnp.c
-    pdo.c
-    fdo.c
-    hardware.c
-    isapnp.h)
+if(ARCH STREQUAL "i386" AND NOT SARCH STREQUAL "xbox")
```

Comment:
Let's create a separate cmake define for it, `ISAPNP_ENABLE` for example. And put it into `sdk/cmake/config.cmake` along with `BUILD_MP` and debugger configuration